docs/common-coreboot-docs/checkbox.md

```diff
@@ -0,0 +1,227 @@
+# Checkbox usage (Ubuntu 22.04)
+
+## Introduction
+
+Checkbox is a system testing platform for Ubuntu. It aims to provide a common
+framework to run all types of tests, from hardware tests, to command line tests,
+unit tests or desktop tests. This document describes the usage of Checkbox on
+Ubuntu 22.04, but it can work on older versions of Ubuntu.
+
+## Prerequisites
+
+To achieve the best results from the tests and do many of them, we should fill
+as many internal and external slots as possible. Before running the checkbox,
+try to fill the slots if the DUT has one:
+
+External:
```

Comment:
Please add new line before enumeration. This will enable the proper visualization.

Reply:
added